specs/src/test/groovy/com/cloudbees/pdk/hen/tests/DeleteBucketContents.groovy

```diff
@@ -0,0 +1,60 @@
+package com.cloudbees.pdk.hen.tests
+
+import spock.lang.Shared
+import spock.lang.Stepwise
+import spock.lang.Unroll
+
+@Stepwise
+class DeleteBucketContents extends PluginTestHelper {
+    @Shared
+    com.cloudbees.pdk.hen.procedures.DeleteBucketContents deleteBucketContents = pluginWithoutConfig.deleteBucketContents
+    @Shared
+    def bucketName = "cbspec"
+    @Shared
+    def folderName = "cbspec-folder"
+
+    def doSetupSpec() {
+        configName = configName.toString().replace("configurations","pluginConfigurations")
+        bucketName += generator( (('a'..'z')+('0'..'9')).join(), 9 )
+        createBucketContents()
+    }
+    def doCleanupSpec() {
+        deleteBucket(bucketName)
+        deleteProject("specs-EC-S3")
+        deleteProject("spec-configs-project-EC-S3")
+    }
+    def "deleteBucketContents"() {
+        when:
+        def r = deleteBucketContents.flush()
+                .config(configName)
+                .bucketName(bucketName)
+                .run()
+        then:
+        assert r.successful
```

Comment:
But was it deleted? There is no assert on that.

Reply:
assertion added in last commit